### PR TITLE
Pop up Window

### PR DESCRIPTION
**Pull request by Sofinaye**
I have added:
A dynamic HTML for the project section using javascript array and object.
A popup window functionality for mobile version.
A popup window functionality for mobile version.
and A functionality for the removal of the popup window.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -540,6 +540,108 @@ html {
...
+.popup-disc-btn-item {
+  padding: 1.5rem;
+  border: none;
+  background-color: #ff6b00;
+  color: #fff;
+  font-family: "Inter", serif;
+  font-size: 17px;
+  font-weight: 700;
+  text-decoration: none;
+}
```
Comment (Gambit142) [inline style.css:639]: - [ ] The **_See Live_** and **_See Source_** button of the popup on _**mobile**_ view is not well positioned. To make it identical to the Figma template chosen, kindly change the value of the `padding` style declaration used on the element. I would suggest you use:
`padding: 1.5rem 1rem`. 👍 

### Your Design

image

### Figma Template

image

File: style.css [hunk 2]
```diff
@@ -540,6 +540,108 @@ html {
...
+
+.popup-disc-btn-item {
+  padding: 1.5rem;
+  border: none;
+  background-color: #ff6b00;
+  color: #fff;
+  font-family: "Inter", serif;
+  font-size: 17px;
+  font-weight: 700;
+  text-decoration: none;
```
Comment (Gambit142) [inline style.css:638]: - [ ] Kindly add the button interactions for the **_See Live_** and **_See Source_** buttons. You can find the style declarations for the different button interactions in this section of the Figma template:

image

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @Sofinaye @Ochiengsteven 👋, 

Good job so far!

## Highlights
- All linters checks passed ✔️ 
- Use of JavaScript to create popup ✔️ 
- Popup is responsive ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:

N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by divinecharlotte — APPROVED** [review]
# Hello @Sofinaye    😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 
### To Highlight 💯

- No linters errors ✔️
- Descriptive PR ✔️
- Descriptive commit messages 👍 





## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**